### PR TITLE
Kiosk cross-fade and double buffering

### PR DESCRIPTION
**Pull request by DelusionalLogic**

### CODE REVIEW

**Comment by VirtualSatai** [comment]
plz pruper tutle

**Comment by DelusionalLogic** [comment]
https://orig00.deviantart.net/d82d/f/2015/213/4/f/swiggity_swag_tutle_by_cantstopimobsessed-d93shsi.png

I think this is pretty correct!

File: kiosk/views.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
+import json
 import time
 from datetime import datetime
 
 from django.http import Http404
 from django.http import HttpResponse
 from django.shortcuts import render
+from django.db.models import Q
```
Comment (VirtualSatai) [inline kiosk/views.py]: isort LUL

File: kiosk/views.py [hunk 2]
```diff
@@ -20,14 +22,21 @@ def find_random_image(request):
...
+        "id": item.id,
+        "url": item.image.url,
+    }
+    return HttpResponse(
+        json.dumps(response_data),
+        content_type="application/json"
+    )
 
 
 def find_next_image(request):
```
Comment (VirtualSatai) [inline kiosk/views.py]: remove

**Pull request merged**